
<launch>

<arg name="map_size_x" default="10.0"/>
<arg name="map_size_y" default="10.0"/>
<arg name="map_size_z" default=" 2.0"/>

  <node pkg ="random_map" name ="random_map_node" type ="random_map_node" output = "screen">    

      <param name="map/x_size"     value="$(arg map_size_x)" />
      <param name="map/y_size"     value="$(arg map_size_y)" />
      <param name="map/z_size"     value="$(arg map_size_z)" />

      <param name="map/circle_num" value="30"/>        
      <param name="map/obs_num"    value="80"/>        
      <param name="map/resolution" value="0.1"/>        

      <param name="ObstacleShape/lower_rad" value="0.1"/>
      <param name="ObstacleShape/upper_rad" value="0.3"/>
      <param name="ObstacleShape/lower_hei" value="1.0"/>
      <param name="ObstacleShape/upper_hei" value="3.0"/>

      <param name="CircleShape/lower_circle_rad"   value="0.6"/>        
      <param name="CircleShape/upper_circle_rad"   value="2.0"/>        
      
      <param name="sensing/rate"   value="0.5"/>        
  </node>
  <!-- if run map node independent-->
   
  <node pkg="waypoint_generator" name="waypoint_generator" type="waypoint_generator" output="screen">
     <remap from="~goal" to="/goal"/>
     <param name="waypoint_type" value="manual-lonely-waypoint"/>    
 </node>
 <node name="rviz" pkg="rviz" type="rviz" args="map.rviz" required="true" /> 
</launch>


